App/Redux/UIRedux.ts

```diff
@@ -52,6 +55,15 @@ const actions = {
   }),
   dismissImagePickerError: createAction('DISMISS_IMAGE_PICKER_ERROR', resolve => {
     return () => resolve()
+  }),
+  refreshMessagesRequest: createAction('REFRESH_MESSAGES_REQUEST', resolve => {
```

Comment:
Was trying to keep any redux actions that query the TextileNode in `TextileNodeRedux`.